dashboard/app/views/pd/regional_partner_contact_mailer/receipt.html.haml

```diff
@@ -1,17 +1,34 @@
-%p
-  Hello
-  = "#{@form[:first_name]}" + ','
+%p= "Hello #{@form[:first_name]},"
+%p Thank you for your interest in supporting computer science education in your region!
 
-%p
-  Thank you for your interest in Code.org's
-  = "#{@interest}" + '. '
-  We will notify you about opportunities in your area when they become available. Below is a confirmation of the information you submitted to us.
+-if @regional_partner
+  %p= "Code.org's Regional Partner, #{@regional_partner.name}, will get back to you."
+-else
+  %p
+    Unfortunately we do not have a local Regional Partner in your region at this time.
+  %p
+    If you are a teacher, please visit our
+    = link_to 'Professional Learning page', CDO.code_org_url('/educate/professional-learning-2018', CDO.default_scheme)
```

Comment:
This should link to https://code.org/educate/professional-learning

Last year we had a year specific page, but this year (as of yesterday's launch) we're back to directing people here.